integrations/Qualaroo/browser.js

```diff
@@ -71,11 +71,19 @@ class Qualaroo {
 
   track(rudderElement) {
     logger.debug("===In Qualaroo track===");
-    const { event } = rudderElement.message;
+    const { message } = rudderElement;
+    const event = get(message, "event");
     if (!event) {
       logger.error("[Qualaroo]:: event is required for track call");
       return;
     }
+
+    const integrationName = get(message, "context.integration.name");
+    if (integrationName === "Qualaroo") {
+      logger.debug(`[Qualaroo]:: dropping callback event: ${event}`);
```

Comment:
@MoumitaM can you confirm if this is the right way to drop callback track events that is coming from Qualaroo itself?

Reply:
Not sure about what the callback track event is but the context doesn't contain an integrations object. And what is this integration.name? there is no name field in integrations object.

Reply:
`context.integration` is custom. Specific for Qualaroo. We are doing the same in our existing integrations `Livechat/Converflow` also. 
I have defined the integration object in the `utils.js` file (line no 4).

Reply:
Qualaroo has some callback events. Which we can use to send data to other enabled destinations.

Reply:
discussed with @MoumitaM we can keep it like this